post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: CONCEPT
 :context: multi-architecture-configuration
 [id="post-install-multi-architecture-configuration"]
-= About clusters with multi-architecture compute machines 
+= About clusters with multi-architecture compute machines
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with x86_64 control plane machines.
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal, {ibmpowerProductName}, and {ibmzProductName} user-provisioned infrastructures with x86_64 control plane machines.
```

Comment:
Per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#azure) Azure needs Microsoft with it.
```suggestion
An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on Amazon Web Services (AWS) or Microsoft Azure installer-provisioned infrastructures and bare metal, {ibmpowerProductName}, and {ibmzProductName} user-provisioned infrastructures with x86_64 control plane machines.
```